locales/fi/actions.json

```diff
@@ -52,7 +52,9 @@
   "actions-status": "Toimenpiteiden tila",
   "actions-status-help": "Aikataulussa olevien toimenpiteiden osuus",
   "actions-updated": "Aktiivisten toimenpiteiden päivitys",
-  "actions-updated-help": "Viimeisen {{count}} päivän aikana päivitettyjen toimenpiteiden osuus",
+  "actions-updated-help-days": "Viimeisen {{count}} päivän aikana päivitettyjen toimenpiteiden osuus",
+  "actions-updated-help-months_one": "Toimenpiteet, jotka on päivitetty viime kuussa",
```

Comment:
@terotik can you suggest better copy for these?

Reply:
Maybe "viime kuussa" -> "viimeisen kuukauden aikana"

Reply:
Oh nice! I'll go with that next time